PWGUD/Tasks/upcTauTau13topo.cxx

```diff
@@ -247,21 +575,22 @@ struct TauTau13topo {
   {
     float delta = p.Phi();
     if (delta < 0)
-      delta += TMath::TwoPi();
+      delta += o2::constants::math::TwoPI;
     if (p1.Phi() < 0)
-      delta -= (p1.Phi() + TMath::TwoPi());
+      delta -= (p1.Phi() + o2::constants::math::TwoPI);
     else
       delta -= p1.Phi();
     if (delta < 0)
-      delta += TMath::TwoPi();
-    if (delta > TMath::Pi())
-      delta = TMath::TwoPi() - delta;
+      delta += o2::constants::math::TwoPI;
+    if (delta > o2::constants::math::PI)
+      delta = o2::constants::math::TwoPI - delta;
     return delta;
```

Comment:
What are you trying to do here?

Reply:
I want to calculate delta phi in a range [0,pi]

Reply:
If there is an easier way to do it, let me know.

Reply:
`RecoDecay::constrainAngle`.

Reply:
Will be resolved in the next occasion